index.js

```diff
@@ -135,7 +135,7 @@ class Schematic {
       let state
       if (versionedMcData.isNewerOrEqualTo('1.13')) {
         state = Object.entries(block.getProperties()).map(([key, value]) => `${key}="${value}"`).join(',')
-        state = state ? ` [${state}]` : ''
```

Comment:
I did some digging and found that the command differs between minecraft editions. Specifically MCPE(bedrock) includes the space while java does not[1]. A simple fix may be to switch on [`versionedMcData.type`](https://github.com/PrismarineJS/node-minecraft-data/blob/master/doc/api.md#type)

[1] https://minecraft.fandom.com/wiki/Commands/setblock#Examples

Reply:
Interesting. I will change the pull request to account for that.